sci-libs/matio/matio-1.5.7.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Library for reading and writing matlab files"
+HOMEPAGE="http://sourceforge.net/projects/matio/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="BSD-2"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples hdf5 sparse static-libs"
+
+RDEPEND="
+	sys-libs/zlib
+	hdf5? ( sci-libs/hdf5 )"
+DEPEND="${RDEPEND}
+	sys-devel/libtool
+	doc? ( virtual/latex-base )"
+
+src_configure() {
+	econf \
+		-docdir="${EPREFIX}/usr/share/doc/${PF}" \
+		$(use_enable hdf5 mat73) \
+		$(use_enable sparse extended-sparse)
+}
+
+src_compile() {
+	emake
+	use doc && 	emake -C documentation pdf
```

Comment:
It's a **take over** from the lower versions. Will remove it in the revised ebuild.